lib/config.js

```diff
@@ -82,6 +83,13 @@ var PromisesDependency;
  *   @return [Boolean] whether to disable S3 body signing when using signature version `v4`.
  *     Body signing can only be disabled when using https. Defaults to `true`.
  *
+ * @!attribute s3DisableTrailingChecksum
```

Comment:
As we discussed last time, can I use `downloadTrailingChecksum` config name instead? On the Node it will default to 'md5' and on browsers this config won't be set.